docs/demo/gaiden-css/scss/base.scss

```diff
@@ -6,6 +6,7 @@
 
 body {
   padding: 20px;
+  font-size: 14px;
```

Comment:
@GeorgeJegon our `font-size` is 16px on `ninja-gaiden.css` file. This can generate some wrong behaviours on components and styles.